setup.py

```diff
@@ -98,7 +98,8 @@ class Test(TestCommand):
 
     def run(self):
         """Run tests."""
-        cmd = f"python3 -m pytest tests/ {self.get_args()}"
+        cmd = "python3 -m pytest tests/ --cov-report term-missing"
```

Comment:
Adding report for missing tested lines. As part of kytos issue [#487](https://github.com/kytos-ng/kytos/issues/487)

Reply:
Great stuff.